disassemblers/ofrak_ghidra/ofrak_ghidra/run_ghidra_server.sh

```diff
@@ -23,5 +42,11 @@ while ! ./server/svrAdmin -add root; do
   echo "Retrying running './server/svrAdmin -add root' ($retry_counter/$max_retries)..."
 done
 
+
 ./server/ghidraSvr restart
-./support/analyzeHeadless . dummy -postScript CreateRepository.java -scriptPath ${OFRAK_GHIDRA_SCRIPTS_PATH} -deleteProject -noanalysis
+./support/analyzeHeadless . dummy -postScript CreateRepository.java $GHIDRA_USER $GHIDRA_PASS $GHIDRA_REPO_HOST $GHIDRA_REPO_PORT -scriptPath ${OFRAK_GHIDRA_SCRIPTS_PATH} -deleteProject -noanalysis
```

Comment:
Shell variables should be quoted when used in case they have spaces.

Probably worth running this script through [`shellcheck`](https://github.com/koalaman/shellcheck) to automatically find potential problems.

```suggestion
./support/analyzeHeadless . dummy -postScript CreateRepository.java "$GHIDRA_USER" "$GHIDRA_PASS" "$GHIDRA_REPO_HOST" "$GHIDRA_REPO_PORT" -scriptPath "${OFRAK_GHIDRA_SCRIPTS_PATH}" -deleteProject -noanalysis
```